src/components/containers/TraceAccordion.js

```diff
@@ -10,78 +10,155 @@ import {Tab, Tabs, TabList, TabPanel} from 'react-tabs';
 const TraceFold = connectTraceToPlot(PlotlyFold);
 
 class TraceAccordion extends Component {
-  render() {
-    const {data = [], localize: _} = this.context;
-    const {
-      canAdd,
-      canGroup,
-      children,
-      messageIfEmptyFold,
-      excludeFits,
-    } = this.props;
+  renderGroupedTraceFolds(groupedTraces) {
+    if (this.props.useFullData) {
```

Comment:
Let's merge `useFullData` and `canGroup` so that you don't need this `if`... mostly so you can remove the `else` case below :)